.werft/observability/monitoring-satellite.ts

```diff
@@ -191,3 +183,29 @@ function ingressReady(namespace: string, name: string): boolean {
     werft.log(sliceName, `${name} ingress not ready.`)
     return false
 }
+
+/**
+ * installSetup creates or replaces everything that is needed to run monitoring-satellite.
+ * This setup includes Custom Resource Definitions(CRDs), kubernetes operators and namespaces.
+ */
+function installSetup() {
+    // Adds a label to the namespace metadata.
+    // This label is used by ServiceMonitor's namespaceSelector, so Prometheus
+    // only scrape metrics from its own namespace.
+    exec('kubectl apply -f observability/monitoring-satellite/manifests/namespace.yaml', {silent: true})
+    exec('kubectl apply -f observability/monitoring-satellite/manifests/podsecuritypolicy-restricted.yaml', {silent: true})
+
+    let crdExists = exec('kubectl get customresourcedefinitions.apiextensions.k8s.io alertmanagers.monitoring.coreos.com', {silent: true}).code == 0
+    if (crdExists) {
+        werft.log(sliceName, "CRDs already exists, replacing CRDs'")
+        exec('for yaml in $(find observability/monitoring-satellite/manifests/prometheus-operator/ -type f -name "*CustomResourceDefinition.yaml"); do cat $yaml | kubectl delete -f -; done', {slice: sliceName})
+        exec('for yaml in $(find observability/monitoring-satellite/manifests/prometheus-operator/ -type f -name "*CustomResourceDefinition.yaml"); do cat $yaml | kubectl create -f -; done', {slice: sliceName})
+    } else {
+        werft.log(sliceName, "CRDs don't exists, creating from scratch")
+        exec('find observability/monitoring-satellite/manifests/prometheus-operator/ -type f -name "*CustomResourceDefinition.yaml" | kubectl create -f -', {slice: sliceName})
+    }
+
+    werft.log(sliceName, 'installing prometheus-operator')
+    exec('for yaml in $(find observability/monitoring-satellite/manifests/prometheus-operator/ -type f ! -name "*CustomResourceDefinition.yaml"); do cat $yaml | kubectl apply -f -; done', {silent: true})
```

Comment:
If CRDs don't have to be installed separately then this can be simplified, see comment above.